### ISSUE TITLE
Bug: Api file output format

### ISSUE THREAD

**Issue by johnfolly** [Cosmo-Tech/Babylon#148]
# API File Output Format

## Current Behavior

The following line in the API get commands converts a snake case formatted map into a camel case token map:

```python
converted_content = convert_keys_case(retrieved_solution, underscore_to_camel)
```

However, it is not working properly, as some tokens are converted while others are not.

## Expected Behavior

All snake case formatted maps should be properly converted into camel case formatted maps.

**Comment by o-reo** [comment]
I'll take a look at it for #141

**Comment by o-reo** [comment]
Hey @johnfolly  can you give me an example file where it does not work ?

**Comment by o-reo** [comment]
Will be fixed in v2.1.0 as I don't use cosmo sdk anymore
